x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterLookupJoinFailuresIT.java

```diff
@@ -112,9 +113,8 @@ public void sendResponse(Exception exception) {
                 assertThat(remoteCluster.getStatus(), equalTo(EsqlExecutionInfo.Cluster.Status.SKIPPED));
                 assertThat(remoteCluster.getFailures(), not(empty()));
                 var failure = remoteCluster.getFailures().get(0);
-                // FIXME: this produces a wrong message currently
-                // assertThat(failure.reason(), containsString(simulatedFailure.getMessage()));
-                assertThat(failure.reason(), containsString("lookup index [values_lookup] is not available in remote cluster [cluster-a]"));
+                assertThat(failure.reason(), containsString(simulatedFailure.getMessage()));
+                assertThat(failure.reason(), containsString("lookup failed in remote cluster [cluster-a] for index [values_lookup]"));
```

Comment:
Prior message looks more specific to me (eg `index is not available in remote cluster` or no index vs `lookup failed in remote cluster` sounds like any failue). Does `simulatedFailure` message cover for that?